crossdc-commons/src/main/java/org/apache/solr/crossdc/common/KafkaMirroringSink.java

```diff
@@ -75,7 +77,9 @@ public void submit(MirroredSolrRequest request) throws MirroringException {
             String message = "Unable to enqueue request " + request + ", configured retries is" + conf.getInt(KafkaCrossDcConf.NUM_RETRIES) +
                 " and configured max delivery timeout in ms is " + conf.getInt(KafkaCrossDcConf.DELIVERY_TIMEOUT_MS);
             log.error(message, e);
-            throw new MirroringException(message, e);
+            if (!ignoreProducerSendError) {
```

Comment:
This will catch all exceptions, right?

Should we only do this for specific cases e.g. when the doc is too big to be submitted?